draft-tschofenig-rats-psa-token.md

```diff
@@ -69,7 +69,7 @@ normative:
     author:
       org: Arm
     title: Platform Security Architecture Firmware Framework 1.0 (PSA-FF)
-    target: https://developer.arm.com/-/media/Files/pdf/PlatformSecurityArchitecture/Architect/DEN0063-PSA_Firmware_Framework-1.0.0-2.pdf
+    target: https://developer.arm.com/documentation/den0063/latest/
```

Comment:
thanks Simon!  Should we use den0063/a/ instead of den0063/latest/ - the latter provides no stability guarantees?

Reply:
Depends whether we want this ref to stay at the original spec or to track updates. I assumed the latter here.

Reply:
I suspect "live" specs are a tad problematic given the fixed / archival nature of RFCs.